store/cart.ts

```diff
@@ -1,118 +1,34 @@
-import { Module } from 'vuex'
-import { CartState } from '../types/Cart'
+export const actions = {
+  async getCart({ rootState, commit }) {
+    const { config } = rootState
 
-const cardsModule: Module<CartState, any> = {
-  state(): CartState {
-    return {
-      type: 'cartWsDTO',
-      appliedOrderPromotions: [],
-      appliedProductPromotions: [],
-      code: '31026397',
-      entries: [
-        {
-          entryNumber: 0,
-          product: {
-            code: '7394376616037',
-            depositPrice: {
-              currencyIso: 'SEK',
-              formattedValue: '0:-',
-              priceType: 'BUY',
-              value: 0.0
-            },
-            keywords: [],
-            name: 'Mellanmjölk Eko',
-            pickPrice: {
-              currencyIso: 'SEK',
-              formattedValue: '11:50',
-              priceType: 'BUY',
-              value: 11.5
-            },
-            unit: {
-              name: 'st'
-            },
-            url:
-              '/handla/varor/mejeri-agg/mjolk-fil/mjolk/mellanmjolk-eko-7300170062642',
-            variances: []
-          },
-          quantity: 1,
-          replace: true,
-          totalPrice: {
-            currencyIso: 'SEK',
-            formattedValue: '11:50',
-            priceType: 'BUY',
-            value: 11.5
-          }
+    let axiosConfig = {}
+    if (config.user !== 'anonymous') {
+      axiosConfig = {
```

Comment:
This is already done in `store/config.ts`

Reply:
Oh, damn, thanks!